config/locales/en.yml

```diff
@@ -108,6 +108,8 @@ en:
       invalid_project_media_archived_value: Sorry, archived value not supported
       invalid_fact_check_language_value: Sorry, language value not supported
       fact_check_empty_title_and_summary: Sorry, you should fill title or summary
+      invalid_feed_saved_search_value: value not supported
+        should belong to a workspace that is part of this feed
```

Comment:
Key missing for this second translation

Reply:
I removed the key as I replaced the validation with `validates_inclusion_of`